seb_openedx/views.py

```diff
@@ -18,18 +19,24 @@ def info_view(request):
     Basic view to show the working version and the exact git commit of the
     installed app
     """
-    try:
-        working_dir = dirname(realpath(__file__))
-        git_data = unicode(check_output(["git", "rev-parse", "HEAD"], cwd=working_dir))
-    except CalledProcessError:
-        git_data = ''
-
-    response_data = {
-        "version": seb_openedx.__version__,
-        "name": "seb_openedx",
-        "git": git_data.rstrip('\r\n'),
-    }
-    return HttpResponse(
-        json.dumps(response_data),
-        content_type="application/json"
-    )
+    if request.user.is_authenticated() and request.user.is_staff:
+        try:
+            working_dir = dirname(realpath(__file__))
+            git_data = unicode(check_output(["git", "rev-parse", "HEAD"], cwd=working_dir))
+        except CalledProcessError:
+            git_data = ''
+
+        middlewares = settings.MIDDLEWARE_CLASSES if hasattr(settings, 'MIDDLEWARE_CLASSES') else settings.MIDDLEWARE
+        is_middleware_installed = 'seb_openedx.middleware.SecureExamBrowserMiddleware' in middlewares
```

Comment:
is it possible to make this depend only of 'seb_openedx.middleware'

Reply:
Done at https://github.com/eduNEXT/seb-openedx/pull/1/commits/4feacb88c069bc1e5836df1e2c143bdb288fb4f3